js/aax.js

```diff
@@ -2175,9 +2166,9 @@ module.exports = class aax extends Exchange {
         //        "ts": 1641026778068
         //    }
         //
-        const positions = this.safeValue (response, 'data');
-        const ts = this.safeInteger (response, 'ts');
-        return this.parsePosition (this.extend (positions[0], { 'ts': ts }));
```

Comment:
@kroitor Why is the timestamp being removed? This is the only place it shows up in the response

Reply:
We should not extend the parsed structures, and we should keep them intact as much as possible. This `ts` thing is not related to positions, it's about the response timestamp, not about positions-timestamps.

Reply:
These positions have an `openTime`. 

Btw, @frosty00 @samgermain how do we define `position['timestamp']` ? 
1. Is it the time when this position was created or opened?
2. Or is it the time when we received this position info?

Reply:
If I had no previous knowledge I would assume that the timestamp corresponded to the time that my position match the data in the response. I would assume that the time the position opened would be called startTime or something

Reply:
Ok, I've restored the timestamp without injecting it into the exchange-specific structures.